packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -50,27 +50,42 @@ export class RenderingScenario extends LitElement {
   }
 
   private toDecibels(rmsDistanceRatio: number) {
-    return (10 * Math.log10(rmsDistanceRatio)).toFixed(2)
+    return 10 * Math.log10(rmsDistanceRatio);
   }
 
   private async loadAnalysis() {
     const analysisPath = `${this.basePath}/analysis.json`;
     this.analysis = await (await fetch(analysisPath)).json();
   }
 
+  private metricTemplate(
+      analysisResults: Array<ImageComparisonAnalysis>, goldenName: string) {
+    const rmsInDecibel =
+        this.toDecibels(analysisResults[0].rmsDistanceRatio).toFixed(2)
```

Comment:
In next PR, im going to remove a lot of things in analysisResults, so it will look nicer then.